plugins/legacy-plugin-chart-sunburst/src/controlPanel.ts

```diff
@@ -51,6 +51,12 @@ export default {
           'When omitted, the color is categorical and based on labels',
       ),
     },
+    color_scheme: {
+      description: t('When only a primary metric is provided, a categorical color scale is used.'),
+    },
+    linear_color_scheme: {
+      description: t('When a secondary metric is provided, a linear color scale is used.'),
+    },
```

Comment:
We _could_ swap this based on the control state when the secondary metric is provided, but maybe that'd be too jarring to the user to just have the color scheme automatically change.